compiler/bindings/c/iree/compiler/dialects/iree_codegen.h

```diff
@@ -66,6 +66,14 @@ MLIR_CAPI_EXPORTED MlirAttribute ireeCodegenCompilationInfoAttrGet(
 MLIR_CAPI_EXPORTED ireeCodegenCompilationInfoParameters
 ireeCodegenCompilationInfoAttrGetParameters(MlirAttribute attr);
 
+MLIR_CAPI_EXPORTED void
+ireeCodegenGetExecutableVariantOps(MlirModule module, int *num_ops,
+                                   MlirOperation *executableOps);
```

Comment:
Let's keep the types consistent with how the other functions handle containers in this file.
```suggestion
ireeCodegenGetExecutableVariantOps(MlirModule module,  size_t *numOps,
                                   MlirOperation *executableOps);
```
also use `thisCase` for function arguments instead of the `snake_case`